e2e/test/scenarios/onboarding/home/homepage.cy.spec.js

```diff
@@ -141,6 +147,69 @@ describe("scenarios > home > homepage", () => {
   });
 });
 
+describe("scenarios > home > custom homepage", () => {
+  describe("setting custom homepage", () => {
+    beforeEach(() => {
+      restore();
+      cy.signInAsAdmin();
+    });
+
+    it("should give you the option to set a custom home page in settings", () => {
+      cy.visit("/admin/settings/general");
+      cy.findByTestId("custom-homepage-setting").findByRole("switch").click();
+
+      cy.findByTestId("custom-homepage-dashboard-setting")
+        .findByRole("button")
+        .click();
+
+      popover().findByText("Orders in a dashboard").click();
+
+      cy.get("main").findByText("Saved");
```

Comment:
We may want to make it a bit narrower than `main` because it is almost as broad as not scoping it. Wdyt about using role="status"?